### PR TITLE
adding a title to noResult

### PR DESCRIPTION
**Pull request by PawelDzi**
I added a title to NoResult. I think it looks good now

### CODE REVIEW

**Review by lkkolosowski — APPROVED** [review]
Nice!

File: src/common/NoResults/styled.js [hunk 1]
```diff
@@ -2,16 +2,19 @@ import styled from "styled-components";
 import { ReactComponent as NoResultsIcon } from "./noResultsIcon.svg"
 
 export const Wrapper = styled.div`
-  margin: 150px auto 0 auto;
+  margin: 0 auto 0 auto;
```
Comment (lkkolosowski) [inline src/common/NoResults/styled.js:5]: good change

Comment (Albrecht-Albi) [inline src/common/NoResults/styled.js:5]: You did a good job of styling the title

File: src/common/NoResults/styled.js [hunk 2]
```diff
@@ -51,4 +54,19 @@ export const StyledIcon = styled(NoResultsIcon)`
...
+  justify-content: left;
+  margin: 0 0 150px;
+  font-size: 36px;
+  font-weight: 600;
+  line-height: 1.2;
+  color: ${({ theme }) => theme.color.woodsmoke};
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
+    font-size: 18px;
+    margin: 0 0 50px;
```
Comment (Albrecht-Albi) [inline src/common/NoResults/styled.js]: You did a good job of mediaquery

**Review by Albrecht-Albi — APPROVED** [review]

**Pull request merged**